README.md

```diff
@@ -1,2 +1,135 @@
+| `image_registry_verify_tls`     | boolean | `true`                                       | Use TLS verification when pushing/pulling images |
+
+### Secret Generation Variables
+| Variable                          | Type   | Default Value                                  | Description                                                                                   |
+| --------------------------------- | ------ | ---------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| `kubeconfig_path`                 | string | None                                           | Full path to kubeconfig file. e.g. `/home/user/.kube/config`                                  |
+| `oc_dir`                          | string | None                                           | Full path to the directory containing the `oc` command binary. e.g. `/home/user/.crc/bin/oc/` |
+| `proteccio_password`              | string | None                                           | Password (SO PIN) used to log into the HSM                                                    |
+| `proteccio_conf_src`              | string | `file:///opt/proteccio/proteccio.rc`           | Full path to the proteccio.rc file                                                            |
+| `proteccio_crt_src`               | string | `/opt/proteccio/certs`                         | Full path to the HSM certificate(s) and client's certificate and key                          |
+| `proteccio_data_secret`           | string | `proteccio-data`                               | Name of the secret used to store client and server certificates                               |
+| `proteccio_data_secret_namespace` | string | `openstack`                                    | Namespace to be used when creating `proteccio_data_secret`                                    |
+| `login_secret`                    | string | `hsm-login`                                    | Name of the secret used to store the password to log into the HSM i                           |
+| `login_secret_field`              | string | `PKCS11Pin`                                    | Secret key used to store the `proteccio_password` data in `login_secret`                      |
+
+## The `proteccio.rc` configuration file
+
+The Eviden Trustway Proteccio HSM reads the configuration file located at `/etc/proteccio/proteccio.rc`.  After running this Ansible role, a default file will be created inside the images of
+Barbican API and Barbican Worker containing dummy data.
+Therefore, before effectively running this role, populate the `/opt/proteccio` directory with a `proteccio.rc` text file having the following layout (remove the <> signs and the texts beginning with #):
+
+```
+[PROTECCIO]
+IPaddr=<IP address of your first HSM device>
+SSL=1
+SrvCert=<name of the crt file corresponding to this HSM's digital certificate>
```

Comment:
Does this need to be a full path?  Or is the assumption that it will be in the same location as the proteccio.rc file?  Ditto for other files referenced here?

Reply:
Yeah, so these will be in the same location as the proteccio.rc file because they will all be mounted as a single secret.  You should make a comment about that.